CHANGES.rst

```diff
@@ -5,6 +5,7 @@ CHANGES for CrateDB Prometheus Adapter
 Unreleased
 ==========
 
+- Add option to configure primary key conflict behaviour.
```

Comment:
I updated them to

```
- Changed the behavior to ignore samples with identical timestamps but different
  values during ingestion. This change aligns with Prometheus' behavior, as
  duplicates should not occur. However, since this enforcement is recent,
  there may be setups with existing duplicates; thus, we now ignore them to
  prevent errors.
```